test/input/beamville/beam.conf

```diff
@@ -161,13 +161,87 @@ beam.outputs.stats.binSize = 3600
 ##################################################################
 # Debugging
 ##################################################################
-beam.debug.debugEnabled = true
+beam.debug.debugEnabled = false
 beam.debug.debugActorTimerIntervalInSec = 60
 beam.debug.actor.logDepth = 12
 
 beam.debug.stuckAgentDetection {
-  enabled = false
-  thresholds = []
+  "checkIntervalMs" : 200,
+  "checkMaxNumberOfMessagesEnabled" : false,
+  "defaultTimeoutMs" : 60000,
+  "enabled" : false,
+  "overallSimulationTimeoutMs" : 100000,
+  "thresholds" : [
```

Comment:
@JustinPihony,
As you can see, it's `JSON`, not `HOCON`. As `HOCON` it should be:
```
beam.debug.stuckAgentDetection {
  enabled = true
  checkIntervalMs = "1000ms"
  checkMaxNumberOfMessagesEnabled = true
  ....
```
So no `:`, no `"` and a bit more things.

Reply:
I was able to get it to work seemingly - testing it more fully now:

```
beam.debug.stuckAgentDetection{
  checkIntervalMs = 200
  checkMaxNumberOfMessagesEnabled = false
  defaultTimeoutMs = 60000
  enabled = true
  overallSimulationTimeoutMs = 100000
  thresholds = [
    {
      actorTypeToMaxNumberOfMessages {
        population = 1
        rideHailAgent = 1
        transitDriverAgent = 1
      }
      markAsStuckAfterMs = 20000
      triggerType = "beam.agentsim.agents.InitializeTrigger"
    }
    {
      actorTypeToMaxNumberOfMessages {
        population = 7
      }
...
```